_config.yml

```diff
@@ -1,6 +1,7 @@
 # [Name of visual theme]
 #theme: jekyll-theme-minimal
 remote_theme: pages-themes/minimal@v0.2.0
+markdown: GFM
```

Comment:
:warning: Add documentation comments (a new line between sections)

```markdown
# [Conversion]
markdown: GFM
```

---

It's a bit weird why use directly `GFM` is a bit different of `kramdown` with `input: GFM` (default). Maybe, to investigate about it and report to Jekyll project @jekyll.

From default config docs: https://jekyllrb.com/docs/configuration/default/
```yml
# Markdown Processors
kramdown:
  auto_ids          : true
  entity_output     : as_char
  toc_levels        : [1, 2, 3, 4, 5, 6]
  smart_quotes      : lsquo,rsquo,ldquo,rdquo
  input             : GFM
  hard_wrap         : false
  footnote_nr       : 1
  show_warnings     : false
```

Reply:
Hmm looks like it's just two different implementations: https://jekyllrb.com/docs/configuration/markdown/ (doc also available on [github](https://github.com/jekyll/jekyll/blob/master/docs/_docs/configuration/markdown.md), which I find easier to read). The two implementations being [kramdown/parser-gfm](https://github.com/kramdown/parser-gfm) vs [github/cmark-gfm](https://github.com/github/cmark-gfm).

Looking at [kramdown/parser-gfm's issue list](https://github.com/kramdown/parser-gfm/issues), it looks like there are some existing differences. I've filed a new issue here: 
[kramdown/parser-gfm#28: Parser doesn't process markdown nested inside of HTML tags](https://github.com/kramdown/parser-gfm/issues/28)

I've added documentation comments to the open PR.